### DISCUSSION TITLE
Is there an option to do not save images?

### DISCUSSION THREAD

**Discussion by gscholer** [gildas-lormeau/SingleFile#977] [General]
Maybe it is weird, but sometimes an image is too large when is included in a html file using base64 code.

**Comment by gildas-lormeau** [comment]
You can block any type of resource via the `Network > blocked resources`, you can also decide to block a resource if it's larger than a given size, via `Network > set maximum size`. Overall, if the size of saved pages are a problem for you, you can have a look at [SingleFileZ](https://github.com/gildas-lormeau/SingleFileZ) which might fulfill your needs.

**Comment by gscholer** [comment]
It worked as expected!

**Comment by KnIfER** [comment]
how to skip save , but preserve (resolve and save) the correct url for images?

**Reply by gildas-lormeau** [reply]
It's not possible (yet), see https://github.com/gildas-lormeau/SingleFile/issues/1217
